objmodel/chapter.md

```diff
@@ -1,86 +1,86 @@
 that of Smalltalk. Smalltalk was an object-oriented programming language
-designed by Alan Kay's group at Xerox Parc in the 1970s. It popularized
-object-oriented programming and is the source of many features found in today's
+designed by Alan Kay's group at Xerox PARC in the 1970s. It popularized
+object-oriented programming, and is the source of many features found in today's
 programming languages. One of the core tenets of Smalltalk's language design
 was "everything is an object". Smalltalk's most immediate successor in use
 today is Ruby, which uses a more C-like syntax but retains most of Smalltalk's
 object model.
 
-The object model of this section will have classes and instances of them, the
+The object model in this section will have classes and instances of them, the
 ability to read
-and write attributes into objects, the ability to call methods on objects and
+and write attributes into objects, the ability to call methods on objects, and
 the ability for a class to be a subclass of another class. Right from the
-beginning, classes will be completely regular objects, that can themselves have
+beginning, classes will be regular objects that can themselves have
 attributes and methods.
-A note on terminology: In this chapter I will use the word "instance" to mean
-"an object that is not a class".
 
-To start the implementation a good approach is to write a test to think about
+A good approach to start with is to write a test to specify
 what the to-be-implemented behaviour should be. All tests presented in this
-chapter will always consist of two parts: A bit of completely regular Python
-code defining and using a few classes, making use of increasingly advanced
-features of the Python object model. The second half of each test is the
+chapter will consist of two parts. First, a bit of regular Python
+code defining and using a few classes, and making use of increasingly advanced
+features of the Python object model. Second, the
 corresponding test using the object model we will implement in this chapter,
-instead of normal Python classes. XXX in the final version these will maybe be
-typeset side by side.
+instead of normal Python classes. 
+<!-- FIXME: Note for @mikedebo: in the final version these will maybe be typeset side by side.-->
 
 The mapping between using normal Python classes and using our object model will
-be done manually in the tests. E.g. instead of writing ``obj.attribute`` in
+be done manually in the tests. For example, instead of writing ``obj.attribute`` in
 Python, in the object model we would use a method ``obj.read_attr("attribute")``.
-This mapping would in a real language implementation be done
-by the interpreter of the language, or a compiler. A further simplification in
-this chapter will be that we make no sharp distinction between the code that
+This mapping would, in a real language implementation, be done
+by the interpreter of the language, or a compiler. 
+
+A further simplification in
```

Comment:
Generally it's good to break up paragraphs as often as you can, especially in technical texts like this. It makes things easier for the reader.